packages/account/src/predicate/predicate.ts

```diff
@@ -117,6 +118,9 @@ export class Predicate<
    */
   sendTransaction(transactionRequestLike: TransactionRequestLike): Promise<TransactionResponse> {
     const transactionRequest = transactionRequestify(transactionRequestLike);
+    if (isTransactionTypeUnknown(transactionRequest)) {
```

Comment:
As above, should this be warning?

Reply:
see [this comment](https://github.com/FuelLabs/fuels-ts/pull/3003#discussion_r1729473790)